Ui/DataProvider/Plan/Collection.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Razorpay\Subscription\Ui\DataProvider\Plan;
+
+use Magento\Framework\View\Element\UiComponent\DataProvider\SearchResult;
+
+/**
+ * Class Collection
+ * @package MageDigest\Grid\Ui\DataProvider\Category\Listing
+ */
+class Collection extends SearchResult
+{
+    /**
+     * Override _initSelect to add custom columns
+     *
+     * @return void
+     */
+    protected function _initSelect()
+    {
+//     SELECT `main_table`.*, `second`.`value`, `third`.`attribute_id` FROM `razorpay_plans` AS `main_table`
+//  LEFT JOIN `catalog_product_entity_varchar` AS `second` ON main_table.magento_product_id = second.entity_id
+//  LEFT JOIN `eav_attribute` AS `third` ON third.attribute_id = second.attribute_id WHERE (third.attribute_code='name' and second.entity_id=main_table.magento_product_id)
```

Comment:
Remove the commented the query and intend the file properly

Reply:
done